addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -164,14 +246,14 @@ def format_alphanumeric(text_to_convert):
             filter_to_apply=lambda l: l['tax_repartition_line_id'].factor_percent >= 0
         )
         for _tax_name, tax_dict in tax_details['tax_details'].items():
```

Comment:
So we should keep using the tax_details as dict (so the old way of working), but also calculating everything for the base_amount instead of just base_amount_currency.  And we can put it in a separate method, but we do not create a new dictionary @dbkosky